booster/library/Booster/Pattern/Rewrite.hs

```diff
@@ -372,7 +372,7 @@ applyRule pat@Pattern{ceilConditions} rule = withRuleContext rule $ runRewriteRu
                     smtUnclear -- no implication could be determined
         Nothing ->
             unless (null unclearRequires) $ do
-                withContext "abort" $
+                withContext "condition" . withContext "abort" $
```

Comment:
I think we use `constraint` instead of `condition` elsewhere in booster and in kore as well
```suggestion
                withContext "constraint" . withContext "abort" $
```